src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -531,6 +538,18 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                     case TokenKind.AtParen:
                     case TokenKind.LParen:
                         {
+                            if (tokenAtCursor.Kind == TokenKind.Equals && lastAst is HashtableAst hashTableAst && CheckForPendingAssignment(hashTableAst))
+                            {
+                                if (lastAst.Parent is DynamicKeywordStatementAst)
+                                {
+                                    // Handle scenarios such as 'configuration foo { File ab { Attributes ='
+                                    bool unused;
+                                    result = GetResultForEnumPropertyValueOfDSCResource(completionContext, string.Empty, ref replacementIndex, ref replacementLength, out unused);
+                                    break;
+                                }
+                                // Handle scenarios such as 'gci | Format-Table @{Label=<tab>'
+                                return null;
+                            }
```

Comment:
This duplicates the call to `GetResultForEnumPropertyValueOfDSCResource`.
How about changing to this:
```
case TokenKind.LParen:
    {
        if (lastAst is AttributeAst)
        {
            ...
        }
        else if (lastAst.Parent is DynamicKeywordStatementAst)  // <-- change here
        {
            ...
        }
        else if (lastAst is HashtableAst hashTableAst && CheckForPendingAssignment(hashTableAst)) // <-- change here
        {
            return result;
        }
        break;
    }
```

Reply:
Second condition doesn't work for `configuration foo { File ab { Attributes =(` test - really `lastAst.Parent` is `CommandExpressionAst`.

Fixed.